wiki/Getting_Started_Contributing_to_Wiki/en.md

```diff
@@ -0,0 +1,182 @@
+
+if you have git installed, put this in your Command Line Interface (CLI) :
+```prolog
+git clone https://github.com/<username>/osu-wiki 
+```
+and this should clone your fork into your specified folder.
+
+Now after making your changes locally, you will need to commit this on your local copy.
+
+this can be done by entering : 
+```prolog
+git add .
+git commit -m "<commit message here>"
+```
+However, in some editors (ie. Visual Studio Code), they provide a Git interface.
+
+![Visual Studio code's Git Interface](10.JPG "Visual Studio's Git Interface")
+
+After Committing your changes, we need to sync this on GitHub, to be able to do this, Enter the following in your Command Line Interface (CLI) :
+
+```prolog
+git push origin <branch>
+```
+It would prompt you to enter your login details, then it should push to GitHub.
+
+_Note: However, If you're authentication details were saved in Git already, no prompt will appear and will proceed directly to push._
+
+## Pull Requests
+
+After making your changes, you must make a "Pull Request" to be able for your changes to be made into the live osu!wiki.
+
+Go to the "Pull Requests" tab.
+
+![Pull Request Tab](6.JPG "Pull Request Tab")
+
+This would lead you to the Pull Requests page.
+
+![PR UI](11.jpg "Pull Requests Page")
+
+This is the Pull Request Page. Now click on "New Pull Request"
+
+![Compare Changes Page](12.jpg "Comparing Changes")
+
+Now we want to compare ``ppy:master`` against your branch.
+
+Now select where did you made your changes in "branch" beside your "head Fork".
+
+![Comparing Changes 2](13.jpg "Comparing Changes 2")
+
+_However if you made changes in your Master Branch, The Changes will be shown without changing anything in the head fork options_.
+
+Now after viewing your changes, click "Create Pull Request" and fill it with the following details:
+
+- **Title** : The Summary of the Changes you've made.
+
+- **Leave a Comment** : the Details of the changes you've made. 
+
+![PR Creation Dialogue](14.jpg "PR Creation Dialogue")
+
+When you are finished, click on the "Create pull request" button and wait for a reviewer to check and, possibly, merge your changes.
+
+Your Changes must follow the [Article Styling Guidelines](/wiki/Article_Style_Guide/) to be able to get merged.
+
+**Your Commit Messages must also explain thoroughly your changes, otherwise, it would not get merged**.
+
+## FAQ
+
+###  Why is the old osu!wiki not staying?
+
+Due to abysmal performance, the old wiki is getting deprecated
+as stated [here](https://github.com/ppy/osu-wiki/issues/69).
+
+###  How would this differ from the old wiki?
+
+- We use GitHub for Collaboration. You can Make mass changes easily and efficiently and you can delay your changes as long as you like.
+- Unlike the old osu!wiki, Changes has to go Pull Requests which are then reviewed by staff or someone with write permissions. This adds a layer of security against spammers and vandals.
```

Comment:
change `Changes` to `changes`
change `Pull` to `pull`
change `Requests` to `requests`